tests/test_configuration.py

```diff
@@ -1,87 +0,0 @@
-def config():
```

Comment:
Removed test cases as we have separate https://github.com/singer-io/tap-twitter-ads/pull/20 for standard tap tester.